libraries/AP_DroneCAN/AP_DroneCAN_DNA_Server.cpp

```diff
@@ -202,8 +202,10 @@ bool AP_DroneCAN_DNA_Server::isValidNodeDataAvailable(uint8_t node_id)
 {
     NodeData node_data;
     readNodeData(node_data, node_id);
+
+    uint8_t empty_hwid[sizeof(NodeData::hwid_hash)] = {0};
```

Comment:
i'm happy with this, but I suspect @peterbarker will insist on removing the = and the 0 in the init, you can do this:

```
  uint8_t xxx[10] {};
```

it generates slightly smaller code

Reply:
It generated identical code for me, but it's probably good to keep the habit so I changed it. I think it matters more for static variables?